enterprise/frontend/src/embedding-sdk/components/public/DashboardCreateModal/use-dashboard-create.ts

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from "react";
+
+import { useSdkDispatch } from "embedding-sdk/store";
+import type { CreateDashboardProperties } from "metabase/dashboard/containers/CreateDashboardForm";
+import Collections from "metabase/entities/collections";
+import Dashboards from "metabase/entities/dashboards";
+import { useStore } from "metabase/lib/redux";
+import type { CollectionId } from "metabase-types/api";
+
+export interface DashboardCreateParameters
+  extends Omit<CreateDashboardProperties, "collection_id"> {
+  collection_id: CollectionId | null;
+}
+
+export const useDashboardCreate = () => {
```

Comment:
If the user tries to call `createDashboard` outside of our Redux context, should we throw a warning? (A sdk user ran into earlier by trying to call the current-user hook outside of the MetabaseProvider context)